src/clientversion.cpp

```diff
@@ -99,5 +100,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
         ss << ")";
     }
     ss << "/";
+    if (!fBaseNameOnly && IsArgSet("-bip148"))
+        ss << "UASF-Segwit:0.3(BIP148)/";
```

Comment:
This user agent will be identical to [the other UASF client](https://github.com/UASF/bitcoin).  Should this be modified to indicate that it is a distinct yet compatible client?